src/nfs/nfs_client_impl.cpp

```diff
@@ -582,32 +579,16 @@ void nfs_client_impl::register_cli_commands()
 {
     static std::once_flag flag;
     std::call_once(flag, [&]() {
-        _nfs_max_copy_rate_megabytes_cmd = dsn::command_manager::instance().register_command(
-            {"nfs.max_copy_rate_megabytes_per_disk"},
-            "nfs.max_copy_rate_megabytes_per_disk [num]",
-            "control the max rate(MB/s) for one disk to copy file from remote node",
-            [](const std::vector<std::string> &args) {
-                std::string result("OK");
-
-                if (args.empty()) {
-                    return std::to_string(FLAGS_max_copy_rate_megabytes_per_disk);
-                }
-
-                int32_t max_copy_rate_megabytes = 0;
-                if (!dsn::buf2int32(args[0], max_copy_rate_megabytes) ||
-                    max_copy_rate_megabytes <= 0) {
-                    return std::string("ERR: invalid arguments");
-                }
-
-                uint32_t max_copy_rate_bytes = max_copy_rate_megabytes << 20;
-                if (max_copy_rate_bytes <= FLAGS_nfs_copy_block_bytes) {
-                    result = std::string("ERR: max_copy_rate_bytes(max_copy_rate_megabytes << 20) "
-                                         "should be greater than nfs_copy_block_bytes:")
-                                 .append(std::to_string(FLAGS_nfs_copy_block_bytes));
-                    return result;
-                }
-                FLAGS_max_copy_rate_megabytes_per_disk = max_copy_rate_megabytes;
-                return result;
+        _nfs_max_copy_rate_megabytes_cmd = dsn::command_manager::instance().register_int_command(
+            FLAGS_max_copy_rate_megabytes_per_disk,
+            FLAGS_max_copy_rate_megabytes_per_disk,
+            "nfs.max_copy_rate_megabytes_per_disk",
+            fmt::format("{}, "
+                        "should be less than 'nfs_copy_block_bytes' which is {}",
+                        max_copy_rate_megabytes_per_disk_desc,
+                        FLAGS_nfs_copy_block_bytes),
+            [](int64_t new_value) -> bool {
+                return new_value == 0 || (new_value << 20) > FLAGS_nfs_copy_block_bytes;
```

Comment:
Could we define a function named something like `check_max_copy_rate_megabytes_per_disk()` to be called by both the command validator and group validator ?